src/NewTools-Debugger-Commands/StDebuggerCommand.class.st

```diff
@@ -103,7 +103,13 @@ StDebuggerCommand >> initialize [
 		stream 
 			<< self class defaultName
 			<< ': '
-			<< self class defaultDescription ])
+			<< self class defaultDescription.
+			self class defaultShortcut ifNotNil: [ :shortCut |
```

Comment:
Could we return a null shortcut object when calling default shortcut in order to  avoid nil checks?

Reply:
Looks like the same work was done in parallel ;)
https://github.com/pharo-spec/NewTools/pull/384

Reply:
Cool :')
We all talked about it on discord a while back, so not surprising.

This could of course be done, but it requires changing a subclassResponsibility.
Could also implement it in every subclasses to return the nil shortcut, but i don't know much about it